meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
 # Explicit machine declaration by build job:
 # MACHINE = "$TARGET_MACHINE"
 
+# Build number shared by all jobs across the different machines:
+# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
+
```

Comment:
Additional thing: this ostroproject-ci.inc file describes bitbake or recipe variables that are overridden for automatic build.
It's not for describing any other variable that might not be available or changed at any time, as those are not written in any config and not expected to be always present.